tests/Issues/NamespacedUse/Fixture/use_without_usage_namespaced.php.inc

```diff
@@ -0,0 +1,14 @@
+<?php
+
+use SomeResponse;
+use AnotherResponse;
+use App\Bar;
+use App\Bar\anyResponse;
+use const App\Bar\Response;
+use const Foo;
+use function App\Bar\response;
+use function add;
+
+?>
+-----
+<?php
```

Comment:
I don't know why closing tag `?>` is deleted here.

Reply:
that's fine, it can be bug or feature of php-parser printer per preferral perspective